pkg/controller/quayecosystem/resources/types.go

```diff
@@ -38,6 +38,8 @@ type QuayConfiguration struct {
 	ValidProvidedQuayConfigPasswordSecret bool
 	QuayImage                             string
 	QuayReplicas                          *int32
+
+	DeployQuayConfiguration bool
```

Comment:
Nitpick: Consistent alignment with types in group above?

Reply:
Updated PR